docs/operating-system/debugging.md

```diff
@@ -64,10 +70,14 @@ docker exec -it homeassistant /bin/bash
 
 ### Generating SSH Keys
 
-Windows instructions for how to generate and use private/public keys with Putty are [here][windows-keys]. Instead of the droplet instructions, add the public key as per above instructions.
-
-Alternative instructions, for Mac, Windows and Linux can be found [here](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#platform-mac).
+On Windows public/private keys can be generate and managed with [Putty][windows-keys]. The public key, that is mentioned in the [Enable SSH host access](#enable-ssh-host-access) section, can be found in the box below _"Public key for pasting into the authorized_keys file"_.
 
-Follow steps 1-4 under 'Generating a new SSH key' (The other sections are not applicable to Home Assistant and can be ignored.)
+On Linux and Mac a new key pair can be generated via:
+```shell
+ssh-keygen -t ed25519 -C "your_email@example.com"
+# ...
+Your identification has been saved in /root/.ssh/id_ed25519 (private key)
+Your public key has been saved in /root/.ssh/id_ed25519.pub (public key)
+```
```

Comment:
I don't think we should provide this, we linked to an extensive guide by GitHub, which is kept up to date as well.

Reply:
I adjusted the section to include both, ssh key generation didn't change that much in the past so it should be fine.